Guesser/FilterTypeGuesser.php

```diff
@@ -77,18 +85,21 @@ public function guessType($class, $property, ModelManagerInterface $modelManager
                 return new TypeGuess('doctrine_orm_date', $options, Guess::HIGH_CONFIDENCE);
             case 'decimal':
             case 'float':
-                $options['field_type'] = 'number';
-
-                return new TypeGuess('doctrine_orm_number', $options, Guess::MEDIUM_CONFIDENCE);
```

Comment:
There is no break statement, which means lines 92 - 96 will be executed, and they do the same. So this was unnecessarily duplicated code if I understand correctly. @deguif : next time, do that in a separate commit (you can use `git add --patch` to do that easily).